### PR TITLE
Use PBR to deploy the application

### PR DESCRIPTION
**Pull request by JoProvost**

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -0,0 +1,28 @@
+[metadata]
+name = python-hostsresover
```
Comment (mat128) [inline setup.cfg]: typo

File: setup.cfg [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+url = https://github.com/internap/python-hostsresover
```
Comment (mat128) [inline setup.cfg]: typo

File: .gitignore [hunk 3]
```diff
@@ -1,4 +1,8 @@
 .idea
 .tox
-python-hostsresolver.iml
-MANIFEST
+.eggs
+*.egg-info
+*.iml
```
Comment (idjaw) [inline .gitignore]: maybe this should go in global ignore so it's not part of the upstream .git file?

**Comment by lindycoder** [comment]
👍

File: setup.cfg [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+[files]
+packages =
+    hostsresolver
+
+[bdist_wheel]
+universal = 1
+
+[nosetests]
+no-path-adjustment = 1
+logging-level = DEBUG
```
Comment (idjaw) [inline setup.cfg]: no newline at end of file. Is this OK for this file?

File: setup.cfg (hunk 2 above)
Comment (JoProvost) [inline setup.cfg]: Argh! Good catch! Thank You!

File: setup.cfg (hunk 1 above)
Comment (JoProvost) [inline setup.cfg]: Argh! Good catch! Thank You!

File: .gitignore (hunk 3 above)
Comment (mat128) [inline .gitignore]: if thats the case, it's the same for the .idea entry at the top.

**Comment by mat128** [comment]
LGTM.

**Comment by mat128** [comment]
+2 +A

File: .gitignore (hunk 3 above)
Comment (JoProvost) [inline .gitignore]: Well, I disagree.  I can't control the global .gitignore of each developer ;)

File: setup.cfg (hunk 4 above)
Comment (JoProvost) [inline setup.cfg]: New line added, thank you

**Pull request merged**